### PR TITLE
Gitleaks/OSV/OWASP CLI arg examples STO-7278-v2

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
### Preview links

- Gitleaks scanner reference > [Additional CLI flags](https://6623cbc2b7c8a1e0145bba2d--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/gitleaks-scanner-reference/#additional-cli-flags)
- OSV scanner reference > [Additional CLI flags](https://6623cbc2b7c8a1e0145bba2d--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/osv-scanner-reference/#additional-cli-flags)
- OWASP scanner reference [Additional CLI flags](https://6623cbc2b7c8a1e0145bba2d--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/owasp-scanner-reference/#additional-cli-flags)

### Merge requirements

- [x] Successful preview build.
- [ ] SME review.
- [ ] Peer review.
- [x] No merge conflicts.
- [x] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/security-testing-orchestration/sto-techref-category/gitleaks-scanner-reference.md [hunk 1]
```diff
@@ -65,6 +65,14 @@ regexes = [
 
 </details>
 
+### Redact secrets
+
+Harness recommends that you configure the Gitleaks step to [redact secrets in the log output](#additional-cli-flags). 
+
+### Speed up your scans
+
+If your scan times are unacceptably high, you can configure the Gitleaks step to [limit the number of commits to scan](#additional-cli-flags). By default, Gitleaks scans the entire commit history.
```
Comment (r0610205) [inline docs/security-testing-orchestration/sto-techref-category/gitleaks-scanner-reference.md:74]: `are unacceptably high` I'd rephrase it. Smth like `take significant time`

File: docs/security-testing-orchestration/sto-techref-category/osv-scanner-reference.md [hunk 2]
```diff
@@ -114,17 +114,14 @@ import StoSettingLogLevel from './shared/step_palette/all/_log-level.md';
...
-
+ 
 Use this field to run the [`osv`](https://google.github.io/osv-scanner/usage/) scanner with flags such as:
 
-`--no-ignore --sbom=/path/to/your/sbom.json`
+`--no-ignore --lockfile=/path/to/your/package-lock.json --lockfile=/path/to/another/Cargo.lock
+`
 
-With these flags, the `osv` binary scans vulnerabilities in the specified SBOM even if it's included in a `.gitignore` file.
+With these flags, the `osv` binary scans vulnerabilities in the specified lockfiles even if they're included in a `.gitignore` file.
```
Comment (r0610205) [inline docs/security-testing-orchestration/sto-techref-category/osv-scanner-reference.md:123]: We need two explanations here. One for -no-ignore, one for lockfiles

Comment (douglas-j-bothwell) [reply docs/security-testing-orchestration/sto-techref-category/osv-scanner-reference.md]: @r0610205 I removed the --no-ignore flag from the example. The OSV doc [mentions a known issue](https://google.github.io/osv-scanner/usage/#ignored-files) related to this flag. I think the two lockfile flags are enough.

File: docs/security-testing-orchestration/sto-techref-category/gitleaks-scanner-reference.md (hunk 1 above)
Comment (douglas-j-bothwell) [reply docs/security-testing-orchestration/sto-techref-category/gitleaks-scanner-reference.md]: To speed up your scan times, you can configure the Gitleaks step to [limit the number of commits to scan](#additional-cli-flags). By default, Gitleaks scans the entire commit history.

**Review by r0610205 — APPROVED** [review]

**Pull request merged**